src/lib/y2partitioner/actions/controllers/lvm_lv.rb

```diff
@@ -271,6 +271,28 @@ def default_size
           lv_type.is?(:thin) ? Y2Storage::DiskSize.GiB(2) : nil
         end
 
+        # The initial value for Y2Partitioner::Dialogs::LvmLvSize::StripesNumberSelector
+        #
+        # @note Since in LVM is not possible to define striping for thin LVs, this is a convenient
+        # method to display the value of their thin pools in the UI.
+        #
+        # @return [Integer, nil] the thin pool stripes number when selected type is a
+        #   thin volume; nil otherwise
+        def default_stripes_number
+          lv_type.is?(:thin) ? thin_pool.stripes : nil
+        end
+
+        # The initial value for Y2Partitioner::Dialogs::LvmLvSize::StripesSizeSelector
+        #
+        # @note Since in LVM is not possible to define striping for thin LVs, this is a convenient
+        # method to display the value of their thin pools in the UI.
+        #
+        # @return [Y2Storage::DiskSize, nil] the thin pool stripes size when selected type is a
+        #   thin volume; nil otherwise
```

Comment:
Is kind of weird that a method called `default_xx` returns `nil` in some circumstances, isn't it?

Reply:
Yes, it is. I was just following the `default_size` approach, since I guess putting there a zero is not an option. Usually, the summary displays no values when there is nothing to show. IMO, those methods are more `initial_xy` than `default_xy`.